### PR TITLE
[EPO-3931] Create TOC for teacher's guide page

### PR DESCRIPTION
**Pull request by georgegomez144**
Update content structure and attrs

[F] Add structure to support sticky-sidenav

### CODE REVIEW

File: templates/teachers/guide.twig [hunk 1]
```diff
@@ -27,237 +34,313 @@
...
+                                        <div class="c-video-embed">
+                                            <iframe id="ytIframe" width="640" height="385"
+                                                src="https://www.youtube.com/embed/{{ relatedVideo.embedCode }}?autoplay=1"
+                                                frameborder="0" allowfullscreen></iframe>
+                                        </div>
+                                    </div>
+                                {% endif %}
+
+                                <div class="c-target-spacer" id="introduction" data-megellan-target="introduction"></div>
+                                <div id="introduction" data-magellan-target="introduction">
```
Comment (blnkt) [inline templates/teachers/guide.twig:63]: double `id`s and `data-magellan-target`s?

File: templates/teachers/guide.twig [hunk 2]
```diff
@@ -27,237 +34,313 @@
...
+                                        {% if entry.investigationTiming | length %}<li><a href="#investigation-timing">Investigation Timing</a></li>{% endif %}
+                                        {% if entry.background | length %}<li><a href="#background">Background</a></li>{% endif %}
+                                        {% if entry.teacherNotes | length %}<li><a href="#teachers-notes">Teachers Notes</a></li>{% endif %}
+                                        {% if entry.investigations | length %}<li><a href="#related-investigations">Related Investigations</a></li>{% endif %}
+                                        {% if entry.ideas | length %}<li><a href="#ideas-for-futher-study">Ideas for Further Study</a></li>{% endif %}
+                                        <li><a href="#common-student-ideas">Common Student Ideas</a></li>
+                                        <li><a href="#differentiation">Differentiation</a></li>
+                                        <li><a href="#standards">Standards</a></li>
+                                    </ul>
```
Comment (blnkt) [inline templates/teachers/guide.twig:224]: - Quick links should update location history just as tabs do, so a user can easily return to the same portion of the page, and so their browser back/forward buttons do what they would expect.
- Active classes like we get with tabs too?

File: templates/teachers/guide.twig [hunk 3]
```diff
@@ -27,237 +34,313 @@
...
+                                <li><a href="#pre-assessment">{{assessment.title}}</a></li>
+                                {%  case "summativeAssessment"  %}
+                                <li><a href="#summative-assessment">{{assessment.title}}</a></li>
+                                {%  case "postAssessment"  %}
+                                <li><a href="#post-assessment">{{assessment.title}}</a></li>
+                                {%  default %}
+                                <li><a href="#assessment">{{assessment.title}}</a></li>
+                                {% endswitch %}
+                                {% endfor %}
+                            </ul>
```
Comment (blnkt) [inline templates/teachers/guide.twig:323]: - Quick links should update location history just as tabs do, so a user can easily return to the same portion of the page, and so their browser back/forward buttons do what they would expect.
- Active classes like we get with tabs too?

**Review by blnkt — CHANGES_REQUESTED** [review]
Screen Shot 2020-10-23 at 12 40 29 PM
- Sidebar should stop being sticky before it interrupts the footer (see screenshot above).  [See this foundation documentation](https://get.foundation/sites/docs/sticky.html#advanced) about how to accomplish using a "bottom" data attribute.

My instinct is to put the sidebar on the lefthand side.  But I would defer to Jose's opinion if you would share a screenshot with him and get his input.

File: src/scss/components.scss [hunk 4]
```diff
@@ -125,6 +131,12 @@
     color: get-color('primary');
 }
 
+.c-target-spacer {
+  display: block;
+  height: 16px * 7;
+  height: 7rem;
+}
+
```
[the remark this answers is not in the document]
Comment (georgegomez144) [reply src/scss/components.scss:139]: There is an attribute `data-offset`, but it doesn't work for me. I've followed their tutorial videos that show how this attribute works, but for some reason it isn't working in this instance. I believed I used this attr for the glossary magellan and that one worked perfectly. My guess is that there are too many "sticky" instances on this page. Either way, the spacer (with the added negative margin) seems to be the correct solution here.

**Review by blnkt — APPROVED** [review]

**Pull request merged**